src/applications/vaos/components/CernerAlert.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import { getCernerURL } from 'platform/utilities/cerner';
+import {
+  selectFeatureStaticLandingPage,
+  selectRegisteredCernerFacilities,
+} from '../redux/selectors';
+
+export default function CernerAlert({ className, pageTitle, level = 2 }) {
+  const H = `h${level}`;
+  const featureStaticLandingPage = useSelector(state =>
+    selectFeatureStaticLandingPage(state),
+  );
+  const cernerFacilities = useSelector(selectRegisteredCernerFacilities);
+  const shouldDisplay = () =>
+    cernerFacilities.length > 0 && pageTitle === 'Appointments';
+
+  return (
+    <>
+      {featureStaticLandingPage &&
+        shouldDisplay() && (
+          <div className={className}>
+            <va-alert status="warning" background-only visible>
+              <H className="vads-u-font-size--h4 vads-u-margin-top--0">
+                To manage appointments at these facilities, go to My VA Health
```

Comment:
Based on the [Design Specs](https://www.figma.com/file/iqjkZx2EbrZu6leL6YfN45/Cerner-Handoff?type=design&node-id=13-33&mode=design&t=81aSZDSCbbtNbSEE-0), when there is a single facility, it should display  "To manage appointments at this facility, go to My VA Health". 

You may want to check with Ciera or Peter about the heading text based on single or multiple facilities in GH61692, Update the static landing page (public websites). The design spec for the alert message in the pubic website reads, "To manage appointments at these facilities, go to My VA Health" regardless of single or multiple facilities. It could be an oversight.

Reply:
Thanks Jenny. Verified and made the change.